### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by quininez**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/hooks/src/use-integration-registry.ts [hunk 1]
```diff
@@ -106,7 +106,14 @@ export function useGetIntegrationReleaseId(
 const getAllIntegrationsIds = async (offset?: number) => {
   const sdk = getSDK();
   const res = await sdk.api.icRegistry.getAllIntegrationsIds(offset);
-  return res.data;
+  let nextIndex = res.data?.nextIndex?._hex;
```
Comment (kenshyx) [inline ui/hooks/src/use-integration-registry.ts]: you can use `res.data?.nextIndex?.toNumber()`

File: ui/hooks/src/use-integration-registry.ts [hunk 2]
```diff
@@ -106,7 +106,14 @@ export function useGetIntegrationReleaseId(
...
+  const integrationIds = res.data?.integrationIds;
+  while (nextIndex && nextIndex !== '0x00') {
```
Comment (kenshyx) [inline ui/hooks/src/use-integration-registry.ts]: you can remove `nextIndex !== '0x00'` when nextIndex is a number(0 by default)

File: ui/hooks/src/use-integration-registry.ts (hunk 1 above)
Comment (quininez) [reply ui/hooks/src/use-integration-registry.ts]: if I cast it to number can I still pass it as offset to the sdk call?

Comment (kenshyx) [reply ui/hooks/src/use-integration-registry.ts]: you can keep it as it is now `const nextRes = await sdk.api.icRegistry.getAllIntegrationsIds(res.data.nextIndex)`

Comment (kenshyx) [reply ui/hooks/src/use-integration-registry.ts]: it uses the response data, not this value from `nextIndex`

Comment (quininez) [reply ui/hooks/src/use-integration-registry.ts]: yes, only that is a bug, it should use the nextIndex

Comment (kenshyx) [reply ui/hooks/src/use-integration-registry.ts]: it should work with the value casted to number

Comment (quininez) [reply ui/hooks/src/use-integration-registry.ts]: but seems it's ok even with the nextIndex passed as a number instead of hex

**Review by kenshyx — APPROVED** [review]

**Pull request merged**